arkouda/pdarraycreation.py

```diff
@@ -699,6 +709,8 @@ def random_strings_uniform(minlen : int, maxlen : int, size : int,
         The number of strings to generate
     characters : (uppercase, lowercase, numeric, printable, binary)
         The set of characters to draw from
+    seed : int
+        Index for where to pull the first returned value
```

Comment:
"Value used to initialize random number generator"

Reply:
updated